infra/charts/feast/charts/feast-jc/values.yaml

```diff
@@ -0,0 +1,154 @@
+# replicaCount -- Number of pods that will be created
+replicaCount: 1
+
+image:
+  # image.repository -- Docker image repository
+  repository: gcr.io/kf-feast/feast-jc
+  # image.tag -- Image tag
+  tag: 0.6.2
```

Comment:
We should add this file to one that is scanned by the `lint versions` script. That ensure that we always keep the versions up to date.